Library/Formula/telegraf.rb

```diff
@@ -1,167 +1,34 @@
-    url "https://github.com/boltdb/bolt.git",
-      :revision => "04a3e85793043e76d41164037d0d7f9d53eecae3"
-  end
-
-  go_resource "github.com/armon/go-metrics" do
-    url "https://github.com/armon/go-metrics.git",
-      :revision => "b2d95e5291cdbc26997d1301a5e467ecbb240e25"
-  end
-
-  go_resource "gopkg.in/dancannon/gorethink.v1" do
-    url "https://github.com/dancannon/gorethink.git",
-      :revision => "8aca6ba2cc6e873299617d730fac0d7f6593113a"
-  end
-
-  go_resource "gopkg.in/Shopify/sarama.v1" do
-    url "https://github.com/Shopify/sarama.git",
-      :revision => "377669e691f8ba3714f32fc2be1915a0207603a7"
-  end
-
-  go_resource "github.com/wvanbergen/kafka" do
-    url "https://github.com/wvanbergen/kafka.git",
-      :revision => "e236a65a9ca6e56d37a2915dc5dab72ac707a4f6"
-  end
-
-  go_resource "github.com/wvanbergen/kazoo-go" do
-    url "https://github.com/wvanbergen/kazoo-go.git",
-      :revision => "15e8d60fd2c1be5b57071b5bbc0b4479cd162a5f"
-  end
-
-  go_resource "github.com/samuel/go-zookeeper" do
-    url "https://github.com/samuel/go-zookeeper.git",
-      :revision => "5bb5cfc093ad18a28148c578f8632cfdb4d802e4"
-  end
-
-  go_resource "github.com/prometheus/client_golang" do
-    url "https://github.com/prometheus/client_golang.git",
-      :revision => "3b16b46a713f181888e5e9a1205ccc34d6917fb9"
-  end
-
-  go_resource "github.com/prometheus/client_model" do
-    url "https://github.com/prometheus/client_model.git",
-      :revision => "fa8ad6fec33561be4280a8f0514318c79d7f6cb6"
-  end
-
-  go_resource "github.com/matttproud/golang_protobuf_extensions" do
-    url "https://github.com/matttproud/golang_protobuf_extensions.git",
-      :revision => "fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a"
-  end
-
-  go_resource "github.com/golang/snappy" do
-    url "https://github.com/golang/snappy.git",
-      :revision => "eaa750b9bf4dcb7cb20454be850613b66cda3273"
-  end
-
-  go_resource "github.com/eapache/queue" do
-    url "https://github.com/eapache/queue.git",
-      :revision => "ded5959c0d4e360646dc9e9908cff48666781367"
-  end
-
-  go_resource "github.com/eapache/go-resiliency" do
-    url "https://github.com/eapache/go-resiliency.git",
-      :revision => "ed0319b32e66e3295db52695ba3ee493e823fbfe"
+  go_resource "github.com/tools/godep" do
+    url "https://github.com/tools/godep.git",
+      :revision => "fe7138c011ae7875d4af21efe8b237f4987d8c4a"
   end
 
   def install
-    ENV["GOBIN"] = bin
     ENV["GOPATH"] = buildpath
-    ENV["GOHOME"] = buildpath
 
     telegraf_path = buildpath/"src/github.com/influxdb/telegraf"
     telegraf_path.install Dir["*"]
 
     Language::Go.stage_deps resources, buildpath/"src"
 
+    cd "src/github.com/tools/godep" do
+      system "go", "install"
+    end
+
     cd telegraf_path do
-      system "go", "build", "-ldflags", "-X main.Version 0.1.3 -X main.Commit 120218f9c6b45bc909c5b9eaa6825d5ace6713be", "cmd/telegraf/telegraf.go"
+      system("$GOPATH/bin/godep go build -o telegraf "\
```

Comment:
Can you split this into separate arguments? This is currently being passed to a shell, which we prefer to avoid; splitting it into separate arguments ensures it bypasses shell splitting. For example:

```ruby
system "#{buildpath}/bin/godep", "go", "build", "-o", "telegraf",
       "-ldflags", "-X main.Version #{version}",
       "cmd/telegraf/telegraf.go"
```